src/screens/questionScreen/QuestionForm.js

```diff
@@ -1,295 +1,310 @@
+   questions.
+   */
+   const renderedQuestions = (questions) => {
+      let questionList = [];
+      for (const q of questions) {
+         const obj = { data: q, setting: question, detail: questionDetail };
+         const dom = renderSingleQuestion(
+            q.answerType,
+            questionProps(q.answerType, obj)
+         );
+         questionList.push(dom);
       }
-    }
+      return <VStack>{questionList}</VStack>;
+   };
 
-    const question = questionData(questionDetail.type)
-    const renderedQuestions = (questions) => {
-        let questionList = []
-        for (const q of questions) {
-            const obj = {data:q, setting: question, detail:questionDetail}
-            const dom = renderSingleQuestion(q.answerType, questionProps(q.answerType, obj));
-            questionList.push(dom)
-        }
-        return (
-            <VStack>
-                {questionList}
-            </VStack>
-        )
-    };
-  const renderSingleSection = (item) => {
-    return (
-        <Accordion.Item>
+   /*
+   Given a section of questions, returns the components to render the section.
+   */
+   const renderSingleSection = (item) => {
+      return (
+         <Accordion.Item>
             <Accordion.Summary>
-                {item.sectionTitle}
-                <Accordion.Icon />
+               {item.sectionTitle}
+               <Accordion.Icon />
             </Accordion.Summary>
             <Accordion.Details>
-                {renderedQuestions(item.questions)}
+               {renderedQuestions(item.questions)}
             </Accordion.Details>
-        </Accordion.Item>
-    )
-  }
+         </Accordion.Item>
+      );
+   };
 
+   /*
+   Returns the components to render all of the sections for the form, given the complete
+   question data object for the form.
+   */
+   const renderSections = (question) => {
+      return (
+         <Accordion index={[0]} mx={4} my={8}>
+            {question.questionsData.map((item) => renderSingleSection(item))}
+         </Accordion>
+      );
+   };
 
-  const renderSections = (question) => {
-    return (
-        <Accordion index={[0]} mx={4} my={8}>
-            { question.questionsData.map(item => renderSingleSection(item)) }
-        </Accordion>
-    )
-  }
-  return (
-    <SafeAreaView>
-      <TopNavigation title={`Questions on ${questionDetail.name}`} backButton navigation={navigation}/>
-      <ScrollView>
-        <Box>
-            {renderSections(question)}
-        </Box>
-      </ScrollView>
-      <Divider />
-      <Box p={4}>
-        <Button onPress={()=> {navigation.goBack()}}>Save Progress</Button>
-      </Box>
-        <VStack alignItems="center" mt={2} mb={8}>
-          <TouchableOpacity onPress={() => Linking.openURL('https://forms.gle/aXVjxVrQU6jm3KUx6')}><Text style={{ color: 'blue' }}>Submit Feedback</Text></TouchableOpacity>
-        </VStack>
-    </SafeAreaView>
-  );
-}
+   return (
+      <SafeAreaView>
```

Comment:
What was the styling that used to be here? Has the view of the questions changed since the styling was deleted?

Reply:
Nope, I don't think it's changed at all